github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml

```diff
@@ -24,21 +24,95 @@ periodics:
         command: [ "/bin/bash", "-ce" ]
         args:
         - |
-          kv_dir=$(cd ../kubevirt && pwd)
+          export GIT_ASKPASS=/usr/local/bin/git-askpass.sh
 
-          ssp_dir=$(cd ../ssp-operator && pwd)
-          ssp_it_path="${ssp_dir}/data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml"
-          ssp_pref_path="${ssp_dir}/data/common-instancetypes-bundle/common-clusterpreferences-bundle.yaml"
+          latest_version=$(curl --fail -s "https://api.github.com/repos/kubevirt/common-instancetypes/releases/latest" | jq -r '.tag_name')
+          description="Automated update of common-instancetypes bundles to ${latest_version}\n\n\\\`\\\`\\\`release-note\nUpdated common-instancetypes bundles to ${latest_version}\n\\\`\\\`\\\`"
 
-          latest_version=$(curl -s https://api.github.com/repos/kubevirt/common-instancetypes/releases/latest | jq -r '.tag_name')
-          base_url="https://github.com/kubevirt/common-instancetypes/releases/download"
-          it_url="${base_url}/${latest_version}/common-clusterinstancetypes-bundle-${latest_version}.yaml"
-          pref_url="${base_url}/${latest_version}/common-clusterpreferences-bundle-${latest_version}.yaml"
+          git-pr.sh -c "cd ../kubevirt && hack/bump-common-instancetypes.sh" -d "echo -e \"${description}\"" -r kubevirt -b update-common-instancetypes -T main
+          git-pr.sh -c "cd ../ssp-operator && hack/bump-common-instancetypes.sh" -d "echo -e \"${description}\"" -r ssp-operator -b update-common-instancetypes -T main
+        resources:
+          requests:
+            memory: "200Mi"
+  - name: periodic-update-common-instancetypes-bundles-release-0.4
+    cron: 0 2 * * *
```

Comment:
nit: Could you stagger the crons so the jobs aren't all kicking off at the same time? The created PRs would have a knock on effect to CI with a lot of jobs being triggered at once.